src/08_comprehensions.py

```diff
@@ -11,13 +11,17 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
+for x in range(5):
```

Comment:
Try to make a list comprehension without a traditional multi-line for loop... it should all take place on the same line